replace-words.py

```diff
@@ -0,0 +1,41 @@
+#Time Complexity : O(n+m)
```

Comment:
Please explain what is m and n here. Also you need to take in account the length of each word as well